### PR TITLE
16 2j registration formgoogle choix

### PR DESCRIPTION
**Pull request by claraxuxu**

### CODE REVIEW

File: app/src/androidTest/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package com.bouboulecorp.thurii.DoubleAuth
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivityTest.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

File: app/src/androidTest/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivityTest.kt [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+import androidx.test.core.app.ActivityScenario
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivityTest.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

File: app/src/androidTest/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivityTest.kt [hunk 3]
```diff
@@ -0,0 +1,96 @@
+package com.bouboulecorp.thurii.EmailRegistration
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivityTest.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

File: app/src/androidTest/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivityTest.kt [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+import androidx.test.core.app.ActivityScenario
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivityTest.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 5]
```diff
@@ -0,0 +1,75 @@
+package com.bouboulecorp.thurii.DoubleAuth
+
+import android.content.Intent
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:3]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.widget.*
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:8]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+            R.layout.spinner_item
+        )
+        val tels = findViewById<Spinner>(R.id.tel_spinner)
+        tel_listener(TelAdapter, tels)
+
+        continueBtnActive()
+    }
+
+    fun goToRegisterMail(view: View) {
+        super.onBackPressed();
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:30]: 🚫 **[ktlint]** <no-semi> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary semicolon

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    fun continueBtnActive() {
+        val tel_number = findViewById<EditText>(R.id.tel_number)
+        val continue_btn = findViewById<Button>(R.id.continueBtn)
+        tel_number.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+
+            override fun beforeTextChanged(s: CharSequence, start: Int,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:39]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline after "("

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:39]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:39]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+                                           count: Int, after: Int) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:40]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (43) (should be 16)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:40]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+                continue_btn.setBackgroundColor(
+                    ContextCompat.getColor(
+                        applicationContext, R.color.custom_color_primary_
+                    )
+                )
+            }
+        })
+    }
+
+    fun tel_listener(TelAdapater:ArrayAdapter<CharSequence>, tels:Spinner) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:52]: 🚫 **[ktlint]** <colon-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ":"

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:52]: 🚫 **[ktlint]** <colon-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ":"

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 11]
```diff
@@ -0,0 +1,70 @@
+package com.bouboulecorp.thurii.EmailRegistration
+
+import android.content.Context
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:3]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+        val checked = findViewById<ImageView>(R.id.checked)
+
+        checked.visibility = View.INVISIBLE
+        continueBtnActive()
+        checkedIconActive(checked)
+    }
+
+    fun goToLastFragment(view: View) {
+        super.onBackPressed();
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:30]: 🚫 **[ktlint]** <no-semi> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary semicolon

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    fun continueBtnActive() {
+        val et_password = findViewById<EditText>(R.id.et_password)
+        val continue_btn = findViewById<Button>(R.id.continueBtn)
+        et_password.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+
+            override fun beforeTextChanged(s: CharSequence, start: Int,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:39]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline after "("

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:39]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:39]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+                                           count: Int, after: Int) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (43) (should be 16)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+            }
+        })
+    }
+
+    fun checkedIconActive(checked: ImageView) {
+        val et_pseudo = findViewById<EditText>(R.id.et_pseudo)
+        et_pseudo.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+
+            override fun beforeTextChanged(s: CharSequence, start: Int,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:57]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline after "("

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:57]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:57]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+                                           count: Int, after: Int) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:58]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (43) (should be 16)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:58]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

**Review by Neyross — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>ktlint</summary>

[app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt|58 col 65|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/64bbefa47512a101687929f4b13422d0158a0a2a/app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt#L58) Missing newline before ")"
[app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt|58 col 66|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/64bbefa47512a101687929f4b13422d0158a0a2a/app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt#L58) Missing newline before ")"
[app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationWay.kt|1 col 1|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/64bbefa47512a101687929f4b13422d0158a0a2a/app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationWay.kt#L1) File must end with a newline (\n)
</details>

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 17]
```diff
@@ -0,0 +1,82 @@
+package com.bouboulecorp.thurii.DoubleAuth
+
+import android.os.Bundle
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    fun continueBtnActive() {
+        val tel_number = findViewById<EditText>(R.id.tel_number)
+        val continue_btn = findViewById<Button>(R.id.continueBtn)
+        tel_number.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+
+            override fun beforeTextChanged(
+                s: CharSequence, start: Int, count: Int, after: Int
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:44]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:44]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:44]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+            ) {}
+
+            override fun onTextChanged(
+                p0: CharSequence?, p1: Int, p2: Int, p3: Int
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:48]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:48]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/DoubleAuth/DoubleAuthActivity.kt:48]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    fun continueBtnActive() {
+        val et_password = findViewById<EditText>(R.id.et_password)
+        val continue_btn = findViewById<Button>(R.id.continueBtn)
+        et_password.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+
+            override fun beforeTextChanged(
+                s: CharSequence, start: Int, count: Int, after: Int
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:40]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+            ) {}
+
+            override fun onTextChanged(
+                p0: CharSequence?, p1: Int, p2: Int, p3: Int
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:44]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:44]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:44]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+        })
+    }
+
+    fun checkedIconActive(checked: ImageView) {
+        val et_pseudo = findViewById<EditText>(R.id.et_pseudo)
+        et_pseudo.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable) {}
+
+            override fun beforeTextChanged(
+                s: CharSequence, start: Int, count: Int, after: Int
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:61]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:61]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:61]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt [hunk 23]
```diff
@@ -0,0 +1,76 @@
...
+            ) {}
+
+            override fun onTextChanged(
+                p0: CharSequence?, p1: Int, p2: Int, p3: Int
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:65]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:65]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/EmailRegistration/RegisterEmailActivity.kt:65]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationWay.kt [hunk 24]
```diff
@@ -0,0 +1,39 @@
+package com.bouboulecorp.thurii.Registration.RegistrationFragment
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationWay.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

**Review by larroyo3 — APPROVED** [review]
Tout bon pour moi

**Pull request merged**